PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -117,5 +117,9 @@
     ],
     "servicesToCheck": "KeyCredential",
     "antiPatternMessage": "Endpoint should not be used with KeyCredential for non-Azure OpenAI clients"
+  },
+  "ImplementationTypeCheck": {
+    "typesToCheck": "implementation",
```

Comment:
This is not really a "type". It's a sub-package name. So, we should consider changing the name here.

Reply:
done - using subPackageName now